ibm_catalog.json

```diff
@@ -965,7 +965,7 @@
                                     "value": "eu-de-2"
                                 },
                                 {
-                                    "displayname": "Madrid 2 (mad02)",
+                                    "displayname": "Madrid 02 (mad02)",
```

Comment:
"displayname": "Madrid 02 DC (mad02) PER",
                                    "value": "mad02"

Sorry my bad. PER was missing. Please add that too. Thank you